src/ChunkedEncoding.h

```diff
@@ -29,12 +29,12 @@
 
 namespace uWS {
 
-    uint32_t STATE_HAS_SIZE = 0x80000000;
-    uint32_t STATE_IS_CHUNKED = 0x40000000;
-    uint32_t STATE_SIZE_MASK = 0x3FFFFFFF;
+    constexpr uint32_t STATE_HAS_SIZE = 0x80000000;
```

Comment:
this was so we could include uWS in two different object files

In one, it's the C bindings

In the other, it's to copy the `Headers` object from JS bindings -> uWS response object

There were duplicate symbol errors without `constexpr` / `inline` / `static`

Reply:
this whole part is already included (but github diff doesn't update)